pkg/manifest/manifest.go

```diff
@@ -170,6 +174,10 @@ func (manifest *manifest) SetNodeRequirements(nodeRequirements map[string]string
 	manifest.NodeRequirements = nodeRequirements
 }
 
+func (manifest *manifest) SetTerminationGracePeriod(minutes int) {
+	manifest.TerminationGracePeriod = minutes
```

Comment:
Where does the documentation belong that we expect minutes rather than seconds to be passed to us?

Reply:
Not documented. Considering the termination grace period granularity, "minute" seems enough, but "second" provides more flexibility and also provides consistency with k8s termination grace period. Would change use "second".